### ISSUE TITLE
Support usage check for formatting directives in f-strings

### ISSUE THREAD

**Issue by jeongukjae** [microsoft/pyright#412]
**Describe the bug**

```python
def return_right_aligned_string():
    some_length = 10
    some_string = "some string to print"

    string_right_aligned = f"{some_string:>{some_length}}"
    return string_right_aligned
```

In above code snippet, `some_length` is used but pyright raised an error. 

`Variable 'some_length' is not accessed pyright (reportUnusedVariable)`

**Screenshots or Code**

<img width="505" alt="스크린샷 2019-12-12 오전 12 22 36" src="https://user-images.githubusercontent.com/8815362/70634408-849f3880-1c75-11ea-874f-71038396ba6a.png">

**VS Code extension or command-line**

both

* version of vscode extension : 1.1.9
* version of cli : 1.1.9

**Comment by erictraut** [comment]
Thanks for the request. This will be fixed in the next version of pyright.

**Comment by erictraut** [comment]
This is now fixed in version 1.1.10, which I just published.

**Comment by jeongukjae** [comment]
@erictraut Thank you!!
